corehq/apps/userreports/pillow.py

```diff
@@ -472,12 +460,6 @@ def __init__(self, processor, pillow_name, topics, num_processes, process_num, r
         # distinguish between doc save errors and data source config errors
         self.retry_errors = retry_errors
 
-    def bootstrap(self, configs=None):
-        self._processor.table_manager.bootstrap(configs)
-
-    def rebuild_table(self, sql_adapter):
-        rebuild_table(sql_adapter)
-
```

Comment:
was this intended in this commit?

Reply:
yes, the commit is called "remove unused methods"

Reply:
oh thanks, i think i read "remove unused imports", my bad.